rules/catch-error-name.js

```diff
@@ -65,7 +65,7 @@ const create = context => {
 				problem.fix = fixer => {
 					const fixings = [fixer.replaceText(node, expectedName)];
 
-					const variables =	scopeManager.getDeclaredVariables(scopeNode);
+					const variables = scopeManager.getDeclaredVariables(scopeNode);
```

Comment:
How did this sneak in

Reply:
White space rules probably aren't looking for tabs inside of lines. They only check leading whitespace.

But definitely suspicious...